azerothcore/azerothcore-wotlk
# fix(Core/RuinsOfAhnQiraj): Ossirian Crystals: Weakness spells

## Changes Proposed:
-  Weakness spells should only affect Ossirian.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/12470

## SOURCE:


## Tests Performed:

- Tested in-game.

## How to Test the Changes:


1. Go to Ossirian, engage and run to a crystal. <Keep in mind the boss should be in distance (30 yds)>
2. Click and wait.
